src/components/Tag/Tag.tsx

```diff
@@ -52,6 +52,7 @@ export const Tag = ({
   icon,
   text,
   hasOutline = false,
+  ...other
```

Comment:
What type does `other` have? Is it `any` or `unknown`?

If so, I wonder if it makes sense to be explicit about max-width and data-testid.

Reply:
In `other` Typescript seems to be unhappy with custom props (i.e. `yee` in this screenshot`) but is ok with props starting with `data-`
![image](https://user-images.githubusercontent.com/86632227/191807948-9fa38656-f005-407f-bb15-18566df0e602.png)

`max-width` will be set either by consumer via classes